pkg/mkimage-raw-efi/install

```diff
@@ -301,6 +327,12 @@ if [ "$INSTALL_ZFS" = true ]; then
    fi
 fi
 
+# In clustered storage config we always install ZFS with raid0 (none)
```

Comment:
Why do you want to hard-code this? Isn't raid0 the default in any case? And if the user wants something else shouldn't the user be able to explicitly set a different raid level argument?